azure-pipelines.sds.pr-release.yml

```diff
@@ -1,185 +1,187 @@
 name: $(Date:yyyyMMddHHmm)-$(Rev:r)
 
 resources:
-    repositories:
-        - repository: azTemplates
-          type: github
-          name: hmcts/azure-devops-templates
-          ref: master
-          endpoint: hmcts
+  repositories:
+    - repository: azTemplates
```

Comment:
as per a previous comment, are there any changes to make to this file? Looks like you might need to change the tab settings in your editor